src/daemons/MetaDaemon.cpp

```diff
@@ -235,6 +236,27 @@ int main(int argc, char *argv[]) {
         }
     }
 
+    {
+        /**
+         *  Only leader part needed.
+         */
+        auto ret = kvstore->partLeader(nebula::meta::kDefaultSpaceId,
```

Comment:
Please check the logic on real cluster

Reply:
> Please check the logic on real cluster

Yes,  I've simple tested two meta nodes on virtual machines. 
I will do complete integration test at later.